tests/unit/client/feedback/dataset/test_base.py

```diff
@@ -37,6 +36,9 @@
 
 
 class TestFeedbackDataset(FeedbackDatasetBase):
+    def update_records(self, records: Union[FeedbackRecord, List[FeedbackRecord]]) -> None:
```

Comment:
As this is unused we can just 
```suggestion
    def update_records(self, **kwargs: Dict[str, Any]) -> None:
```